src/core/common/instance.hpp

```diff
@@ -221,7 +221,7 @@ class Instance : public otInstance, private NonCopyable
     void SetLogLevel(otLogLevel aLogLevel)
```

Comment:
How about if we change `SetLogLevle()` as well to be a `static` method? and change its uses?

Reply:
Done!